### PR TITLE
Fix/#141

### LINKED ISSUE #141: Single line text  /  Texto De Una Sola Línea
** Explanation:**
The text below the title should be displayed on a single line as shown in the original.

**Current result:**
Being in the application on page 2, for the 320x633 resolution the text under the title is seen in two lines.

**Expected result:**
That being in the application on page 2, for the 320x633 resolution the text under the title is seen in a single line.

*Screen size:*
320x633 movil.
1024x633 portatil.

### Screenshots:
Original:
image

Our:
image
image

### Check list:
This issue can be closed when the following tasks are completed:
-[Verify that the text for the specified resolution is displayed in a single line]
### References:
Original:
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/maps-proyect?node-id=407%3A46
Our:
 https://github.com/Azordev/maps-project

** Explicación:**
El texto debajo del título debe mostrarse en una sola línea como se muestra en el original.

**Resultado actual:**
Estando en la aplicación en la página 2, para la resolución 320x633 se ve el texto debajo del título en dos líneas.

**Resultado Esperado:**
Eso siendo en la aplicación en la página 2, para la resolución de 320x633 el texto debajo del título se ve en una sola línea.

*Tamaño de pantalla:*
320x633 móvil.
1024x633 portátil.

### Capturas de pantalla:
Original:
imagen
Nuestra:
imagen
imagen

### Lista de Verificación:
Este problema se puede cerrar cuando se completan las siguientes tareas:
-[Verificar que el texto para la resolución especificada se muestra en una sola línea]
### Referencias:
Original:
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/maps-proyect?node-id=407%3A46
Nuestro:
https://github.com/Azordev/maps-project

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
### Description

 Single line text / Texto De Una Sola Línea #141 

#### Screenshots

fix141#1
fix141#2

### CODE REVIEW

**Comment by angelik0828** [comment]
perform verification, the change is present in the different screen sizes, also in desktop view.

1. the text size adapts and looks good on screen sizes from mobile to desktop.

image
image
image
image

**Review by angelik0828 — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

File: src/pages/Delivery/Delivery.styled.jsx [hunk 1]
```diff
@@ -96,6 +96,10 @@ const HeaderSubTitle = styled.div`
   line-height: 28px;
   font-family: 'Poppins', sans-serif;
 
+  @media (max-width: 768px){
+    justify-content: left ;
```
Comment (heliomar-pena) [inline src/pages/Delivery/Delivery.styled.jsx:100]: please check indentation

```suggestion
    justify-content: left; 
```

File: src/pages/Delivery/Delivery.layout.jsx [hunk 2]
```diff
@@ -62,16 +62,16 @@ const Delivery = ({
...
             txt={clientAddress} 
-            style={{ flexDirection: 'row',fontWeight:'400',fontSize:'13px'}}
+            style={{ flexDirection: 'row',fontWeight:'600',fontSize:'13px'}}
           >
             <img src={gps} width="22px" height="31px" alt="Gps icon" style={{ margin: '10px'}} />
           </FooterMapAddress>
-          <FooterMapAddress cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
-            <Text medium bold margin="8" style={{ textTransform: 'lowercase', color:'#003B62',fontWeight:'700'}}>
+          <FooterMapAddress style={{ fontSize: '25px', lineHeight: '2.0'}} cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
+          <Text medium bold margin="8" style={{ textTransform: 'lowercase', fontSize: '23px',  color: '#003B62', fontWeight: '700' }}>
```
Comment (heliomar-pena) [inline src/pages/Delivery/Delivery.layout.jsx:70]: Is not possible to use class instead of inline-style?

File: src/pages/Check/index.jsx [hunk 3]
```diff
@@ -51,7 +51,7 @@ const Check = () => {
   return (
     <Layout
       headerTitle="Hola!"
-      headerSubtitle="Ingresa tu número de boleta"
+      headerSubtitle="Inserte su numero de Boleta"
```
Comment (heliomar-pena) [inline src/pages/Check/index.jsx:54]:
```suggestion
      headerSubtitle="Inserte su número de Boleta"
```

**Review by heliomar-pena — COMMENTED** [review]
good work! some comments

**Pull request merged**